doxygen does not build new changes file

I observed that the deal.II installation does not build the file `changes_after_8_5_0.html` file any more. As a consequence, the link on the homepage (Dev -> Recent changes) to https://www.dealii.org/developer/doxygen/deal.II/changes_after_8_5_0.html is broken. I do not have the time to look into it right now, but it is broken since at least May 27 (that's the last time I have an update on that file for a local documentation build).

I think it is because of this error:
```
/home/bangerth/p/deal.II/1/build-doc/doc/news/changes.h:721: warning: reached end of file while inside a f$ block!
```
I'll look into this tonight if nobody beats me (but have to run now to sell deal.II ;-) ).

--- a/doc/news/changes/minor/20170529DenisDavydov
+++ b/doc/news/changes/minor/20170529DenisDavydov
@@ -1,6 +1,6 @@
 New: deal.IIFeatureConfig.cmake will contain detailed information about
 feature configuration and is installed along with other CMake files.
-These variables can be loaded by calling INCLUDE(${DEAL_II_FEATURE_CONFIG}).
+These variables can be loaded by calling INCLUDE(\${DEAL_II_FEATURE_CONFIG}).
 This is useful when users want to link in additional parts of dependencies, not
 required by deal.II (i.e. filesystem library from Boost package).
 <br>

